coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -152,9 +172,15 @@ def load(cls, language: str, docstyle: str, coalang_dir=None):
             raise KeyError("Language {!r} is not defined for docstyle {!r}."
                            .format(language, docstyle))
 
+        metadata_settings = ("param_start", "param_end", "return_sep")
+
+        metadata = cls.Metadata(*[str(docstyle_settings.get(req_setting, ""))
+                                  for req_setting in metadata_settings])
```

Comment:
you don't need the list `[]`, unpacking generators is also possible :)
so just replace the `[]` with `()`